e2e/cypress/integration/system_console/compliance/custom_terms_of_service_spec.js

```diff
@@ -9,32 +9,43 @@
 // Group: @system_console
 
 describe('Custom Terms of Service', () => {
-    const customTermsOfServiceText = 'Test custom terms of service';
     let testUser;
     let testTeam;
 
     before(() => {
-        cy.apiUpdateConfig({
-            EmailSettings: {
-                RequireEmailVerification: true,
-            },
-            SupportSettings: {
-                CustomTermsOfServiceEnabled: true,
-            },
-        });
-
-        cy.apiCreateTermsOfService(customTermsOfServiceText);
-
         cy.apiInitSetup().then(({team, user}) => {
             testUser = user;
             testTeam = team;
         });
     });
 
     it('MM-T1190 - Appears after creating new account and verifying email address', () => {
+        const customTermsOfServiceText = 'Test custom terms of service';
+
         // # Verify new user email
         cy.apiVerifyUserEmailById(testUser.id);
 
+        // # Login as admin
+        cy.apiAdminLogin();
+
+        cy.apiUpdateConfig({
+            EmailSettings: {
+                RequireEmailVerification: true,
+            },
+        });
+
+        // # Visit custom terms of service page
+        cy.visit('/admin_console/compliance/custom_terms_of_service');
+
+        // # Enable custom terms of service
+        cy.findByTestId('SupportSettings.CustomTermsOfServiceEnabledtrue').click();
+
+        // # Set the terms of service to the first value
+        cy.findByTestId('SupportSettings.CustomTermsOfServiceTextinput').clear().type(customTermsOfServiceText);
+
+        // # Save config
+        cy.get('#saveSetting').click();
+
```

Comment:
@saturninoabril I modified this test to have the sysadmin go through the UI as the api command was resulting in the test being flakey at times, im not sure what the difference is of going through the UI vs API but instead of digging in to figure that out I decided to just go the more consistent route of using the UI, its better this way since it also tests the system console functionality which it previously wasnt